pkg/review/commit_review_status.go

```diff
@@ -232,7 +232,7 @@ func getApprovalStatus(request *PullRequest) string {
 // issues for commits that do not have the status GithubPRApproved.
 func processReviewStatus(ctx context.Context, fetcher BreakGlassIssueFetcher, cfg *Config, commitReviewStatus CommitReviewStatus) *CommitReviewStatus {
 	logger := logging.FromContext(ctx)
-	logger.InfoContext(ctx, "processing commitReviewStatus", "commit_review_status", commitReviewStatus.SHA)
+	logger.InfoContext(ctx, "processing commitReviewStatus", "commitReviewStatus", commitReviewStatus)
```

Comment:
Lint apparently disagrees with you on using the snake_cases

Reply:
fixed.